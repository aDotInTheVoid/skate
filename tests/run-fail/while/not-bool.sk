fn main() {
    while 2 + 4 {
        // Empty
    }
}
