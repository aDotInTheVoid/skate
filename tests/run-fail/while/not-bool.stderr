error: Expected `bool`, got `int`
  ┌─ $DD/tests/run-fail/while/not-bool.sk:2:11
  │
2 │     while 2 + 4 {
  │           ^^^^^ Evaluated to `6`

